internal/pkg/cache/add.go

```diff
@@ -327,6 +327,8 @@ type AddOpts struct {
 	Username string
 	// Optional password for basic auth to a registry that requires authentication.
 	Password string
+	// Temporary pack location that we want to copy it from.
+	CurrentLocation string
```

Comment:
I know the other parameters also do this, but comments should start with the parameter name.

Reply:
Good catch, but this field is actually obsolete, it's not needed since we don't support copying to cache anymore. I'll remove it.